assets/pesy-package.template.json

```diff
@@ -13,17 +13,13 @@
   "buildDirs": {
     "test": {
       "require": ["<PUBLIC_LIB_NAME>"],
-      "main": "Test<PACKAGE_NAME_UPPER_CAMEL>",
-      "name": "Test<PACKAGE_NAME_UPPER_CAMEL>.exe"
+      "bin": "Test<PACKAGE_NAME_UPPER_CAMEL>.re as Test<PACKAGE_NAME_UPPER_CAMEL>.exe"
```

Comment:
Can we use
```
{
  "bin": {
    "Test<PACKAGE_NAME_UPPER_CAMEL>.exe": "Test<PACKAGE_NAME_UPPER_CAMEL>.re"
  }
}
```
instead? I think this is more familiar to npm users.

Reply:
Also I think this will simplify implementation as it won't be required to have a parser for this little syntax if we encode such structure with JSON instead.

Consider also the case where one needs to "hack" with pesy config using some adhoc script — it's just easier when everything in JSON.

Another point is that `x as y` can be confusing while something like:
```
{bin: 'x', name: 'y'}
```
or
```
{bin: 'x', as: 'y'}
```
provides more info on what `x` and `y` mean.

Reply:
@andreypopp 

> Also I think this will simplify implementation as it won't be required to have a parser for this little syntax if we encode such structure with JSON instead. 

> Consider also the case where one needs to "hack" with pesy config using some adhoc script — it's just easier when everything in JSON

Super :+1: 

> {bin: 'x', name: 'y'}

I like the first proposal better. 

```json
"bin": {
  "Test<PACKAGE_NAME_UPPER_CAMEL>.exe": "Test<PACKAGE_NAME_UPPER_CAMEL>.re"
}
```
Many of us are already familiar that key represents the name of the binary and the value is the file that creates it. Explicit labels `bin` and `as` feel unnecessary.

This is my opinion. I would love to hear others' too.

cc @jordwalke @jaredly

Reply:
Yeah that sounds great to me!